### ISSUE TITLE
[Controller Request] Wii Remote w/ MotionPlus (as an adapter or built in)

### ISSUE THREAD

**Issue by spudpiggy** [isXander/Controlify#152]
**Controller name**

Wii Remote / Wii Remote Plus

**Conroller VID & PID**

For my **non plus Wii Remote**, the VID is 057E and the PID is 0306, so the paired values are `057e:0306`.
However, later Wii Remote Pluses apparently have a different PID? I don't have one of those to test tho.

Thing is, the Wii Remote doesn't show up as a normal controller, at least on Windows without custom drivers.
Hid Wiimote exists, but it doesn't support the extra axes provided by the MotionPlus.
I also hear later Pluses can be harder to pair or something? IDK, pairing is janky even on non plus cus it doesnt remember to stay paired to your computer. Luckily, once you've paired it once through the **old** devices dialog, you can use Dolphin to quickly resync it again.

tl;dr wii remote support would be dope, but it'd also be kinda hard - hid wiimote exists but doesnt use motionplus

**Comment by 31j** [comment]
I own a Wii Motion Plus controller (the mario edition one from I think the Wii U era). What information would I need to provide here so both of the OG and the Wii Motion Plus can be added? I think I also have access to an earlier-ish Wii Remote Plus, as I have a friend who has one that would have come with the Wii Mini (the ugly red and black wii with no internet)

**Comment by spudpiggy** [comment]
give the VID and the PID - google how to get those, but if you're on windows you can get it from device manager once you pair it up.
About pairing - it might be different for the plus, but my non plus needs to be paired via the old devices and printers menu in control panel (i use windows 10)
I don't enter a pin, I just hit next to skip that screen.
Mine shows as "Nintendo RVL-CNT" but i think plus is "Nintendo RVL-CNT-TR", and I have to pair it every time. Yes, i use the sync button. (Can someone tell me how to keep it paired on Windows 10?)

I also heard the plus can have other issues, like how HBC setup doesn't work with them? Can't weigh in though cus I've never even used a wii remote plus

**Comment by ff66theone** [comment]
Well in the mean time I found a userspace driver for Linux that does 50% of what I needed (accelerometer and Nunchuk support) but still isn't able to use IR or MotionPlus (I assume due to the difficulty of the implementation).
